torch/utils/data/_utils/pin_memory.py

```diff
@@ -45,18 +45,30 @@ def _pin_memory_loop(in_queue, out_queue, device_id, done_event):
         del r  # save memory
 
 
-def pin_memory(data):
+def pin_memory(data, device=""):
```

Comment:
is device="" an idiom that we use in other places? or is device=None better?

Reply:
should be  `device=None` as more pythonic